agent/structs/config_entry.go

```diff
@@ -741,6 +766,13 @@ func (c *ConfigEntryRequest) UnmarshalBinary(data []byte) error {
 	return nil
 }
 
+func setID(meta *map[string]string, id string) {
```

Comment:
Oh it looks like you already did a variation of my `setSafely` but didn't really use it all the way?

Reply:
I don't remember why I ended up not using it 😅